src/components/AnnualReport/AGMDate.vue

```diff
@@ -1,68 +1,114 @@
-              label="Annual General Meeting Date"
-              placeholder="Select your Annual General Meeting Date"
-              prepend-icon="mdi-calendar"
-              v-on="on"
-              readonly
-              filled
+      <div>
+        <v-form ref="form" class="value date">
+          <!-- date picker -->
+          <v-menu
+            ref="menu"
+            v-model="menu"
+            :close-on-content-click="false"
+            nudge-right="50"
+            transition="scale-transition"
+            max-width="290"
+          >
+            <template v-slot:activator="{ on, attrs }">
+              <v-text-field
+                data-test-id="agm-date-text"
+                class="pt-0"
+                v-model="dateText"
+                :disabled="noAgm"
+                :rules="agmDateRules"
+                label="Annual General Meeting Date"
+                placeholder="Select your Annual General Meeting Date"
+                append-icon="mdi-calendar"
+                readonly
+                v-bind="attrs"
+                v-on="on"
+                filled
+              />
+            </template>
+            <v-date-picker
+              data-test-id="agm-date-picker"
+              v-model="datePicker"
+              :min=arMinDate
+              :max=arMaxDate
+              no-title
+              @input="menu=false"
+              @change="onDatePickerChanged($event)"
             />
-          </template>
-          <v-date-picker
-            data-test-id="agm-date-picker"
-            v-model="datePicker"
-            :min=minDate
-            :max=maxDate
-            no-title
-            @input="menu=false"
-            @change="onDatePickerChanged($event)"
-          />
-        </v-menu>
-
-        <!-- only display this info if a date has been selected -->
-        <div class="validationErrorInfo" v-if="dateText">
-          <span v-if="!allowCOA && !allowCOD">
-            You can not change your Registered Office Addresses or Directors in this Annual Report because your AGM
-            predates another filing that may have conflicting changes.
-          </span>
-          <span v-else-if="!allowCOA">
-            You can not change your Registered Office Addresses in this Annual Report because your AGM predates another
-            filing that may have conflicting changes.
-          </span>
-          <span v-else-if="!allowCOD">
-            You can not change your Directors in this Annual Report because your AGM predates another
-            filing that may have conflicting changes.
-          </span>
-        </div>
-      </v-form>
+          </v-menu>
+
+          <!-- restriction messages -->
+          <!-- date must have been selected first -->
+          <div v-if="dateText" class="restriction-messages mt-4">
+            <span v-if="!allowCOA && !allowCOD">
+              You cannot change your Registered Office Addresses or Directors in this Annual Report because your
```

Comment:
https://www.grammarly.com/blog/cannot-or-can-not/#:~:text=Here%20is%20a%20quick%20summary,.%20.%20.%20but%20also.%E2%80%9D